src/common/fetch.ts

```diff
@@ -132,3 +148,27 @@ export const fetchComment = async (supabase: Supabase): Promise<string> => {
 
     return data.value;
 };
+
+export type PackingSlotsLabelsAndValues = [string, Schema["packing_slots"]["primary_key"]][];
+
+export const fetchPackingSlotsInfo = async (
+    supabase: Supabase
+): Promise<PackingSlotsLabelsAndValues> => {
+    const { data, error } = await supabase
+        .from("packing_slots")
+        .select("primary_key, name")
+        .order("order")
+        .eq("is_shown", true);
+
+    if (error) {
+        const logId = await logErrorReturnLogId("Error with fetch: Packing slots data", error);
+        throw new DatabaseError("fetch", "packing slots data", logId);
+    }
+
+    const packingSlotsLabelsAndValues: PackingSlotsLabelsAndValues = data!.map((packingSlot) => [
```

Comment:
I don't think we need the `!`
```suggestion
    const packingSlotsLabelsAndValues: PackingSlotsLabelsAndValues = data.map((packingSlot) => [
```